default.nix

```diff
@@ -0,0 +1,50 @@
+#
+# Nix Setup for Hyrise
+#
+# First install Nix (see https://nixos.org/manual/nix/stable/installation/).
+# Then in the root of this repository, run 
+#
+#   nix-shell --pure`
+#
+# The `--pure` flag ensures that the shell does not take over environment
```

Comment:
How is "environment variables from the host" connected to "all software installed"?
Is the software installation dependent on same env vars?

Reply:
Yes! Usually, a dependency is found by being referenced by an Env Var. If you unset it, you usually end up with an unusable binary somewhere in the system. I am not aware of how cmake finds their dependencies, but I am very confident it also requires variable in some cmake or path variables.

e.g., for the Nix shell here, we end up with the following env configuration (running `nix-shell --pure && env`):

<details>

<summary>Nix System</summary>

```bash
SHELL=/nix/store/h4fzifvim9ls3v6jf2l41xsd0mn69pzw-bash-interactive-5.2p26/bin/bash
hardeningDisable=format fortify fortify3
NIX_BUILD_CORES=10
configureFlags=
mesonFlags=
PYTHONNOUSERSITE=1
CMAKE_FRAMEWORK_PATH=/nix/store/9rgb0lr5i4vbizzhac5b6fk169n8mrkk-apple-framework-CoreFoundation-11.0.0/Library/Frameworks:/nix/store/9rgb0lr5i4vbizzhac5b6fk169n8mrkk-apple-framework-CoreFoundation-11.0.0/Library/Frameworks:/nix/store/9rgb0lr5i4vbizzhac5b6fk169n8mrkk-apple-framework-CoreFoundation-11.0.0/Library/Frameworks:/nix/store/9rgb0lr5i4vbizzhac5b6fk169n8mrkk-apple-framework-CoreFoundation-11.0.0/Library/Frameworks
shell=/nix/store/s30jrpgav677fpc9yvkqsib70xfmx7xi-bash-5.2p26/bin/bash
depsHostHost=
__sandboxProfile=
PYTHONHASHSEED=0
STRINGS=strings
CMAKE_INCLUDE_PATH=/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/include:/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/include:/nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev/include:/nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev/include:/nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev/include:/nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev/include:/nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev/include:/nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev/include:/nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev/include:/nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev/include:/nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev/include:/nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev/include:/nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev/include:/nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev/include:/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/include:/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/include:/nix/store/mhmq1fci2kw6jp6qjv9hnryx147d8zpr-libcxx-16.0.6-dev/include:/nix/store/mhmq1fci2kw6jp6qjv9hnryx147d8zpr-libcxx-16.0.6-dev/include:/nix/store/flagrdvds9knmjba8c4r9wqhj46s7xdb-compiler-rt-libc-16.0.6-dev/include:/nix/store/flagrdvds9knmjba8c4r9wqhj46s7xdb-compiler-rt-libc-16.0.6-dev/include:/nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/include:/nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/include:/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/include:/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/include:/nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev/include:/nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev/include:/nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev/include:/nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev/include:/nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev/include:/nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev/include:/nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev/include:/nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev/include:/nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev/include:/nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev/include:/nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev/include:/nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev/include:/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/include:/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/include:/nix/store/mhmq1fci2kw6jp6qjv9hnryx147d8zpr-libcxx-16.0.6-dev/include:/nix/store/mhmq1fci2kw6jp6qjv9hnryx147d8zpr-libcxx-16.0.6-dev/include:/nix/store/flagrdvds9knmjba8c4r9wqhj46s7xdb-compiler-rt-libc-16.0.6-dev/include:/nix/store/flagrdvds9knmjba8c4r9wqhj46s7xdb-compiler-rt-libc-16.0.6-dev/include:/nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/include:/nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/include
depsTargetTarget=
stdenv=/nix/store/ashgigyp2dkvfi1spyl9nsnirvbk67vh-stdenv-darwin
builder=/nix/store/s30jrpgav677fpc9yvkqsib70xfmx7xi-bash-5.2p26/bin/bash
MACOSX_DEPLOYMENT_TARGET=11.0
PWD=/Users/robertrichter/Git/hyrise-histograms
NIX_PROFILES=/nix/var/nix/profiles/default /Users/robertrichter/.nix-profile
SOURCE_DATE_EPOCH=315532800
LOGNAME=robertrichter
NIX_ENFORCE_NO_NATIVE=1
__propagatedSandboxProfile=
CXX=clang++
TEMPDIR=/tmp
system=aarch64-darwin
HOST_PATH=/nix/store/47zwbmparr2g3b0hci44vma95k1j8a3v-coreutils-9.5/bin:/nix/store/dj4z6gimsxzlw0b4dnpcgv4b6wxijx13-findutils-4.9.0/bin:/nix/store/7cjp56110icrph79302pmvv980c6cxw1-diffutils-3.10/bin:/nix/store/qxhcgmw2va7wdqz551bg1wn2h0m5ggy4-gnused-4.9/bin:/nix/store/9rf817iskvk4m8bph69q01q5pqbzc551-gnugrep-3.11/bin:/nix/store/i6gxar94g2iiid6v54hlnz9pipp6p950-gawk-5.2.2/bin:/nix/store/vd6i09s9gbb6d9dgvcwkhd7h680laayv-gnutar-1.35/bin:/nix/store/dl9rjapvkcnypn01cvpbns1xxi4kp5qq-gzip-1.13/bin:/nix/store/26a0s1d6vlbpvg1x59q3jjgdg51i99kc-bzip2-1.0.8-bin/bin:/nix/store/qkkliqj6qhzlkchlkqc6n2iry22gi78q-gnumake-4.4.1/bin:/nix/store/s30jrpgav677fpc9yvkqsib70xfmx7xi-bash-5.2p26/bin:/nix/store/q0c9g6cywh0n40vapnl2f4s0bq4fkh59-patch-2.7.6/bin:/nix/store/q040nslh604a99yvk3qa1avgw5sgmzda-xz-5.4.6-bin/bin:/nix/store/xy7w9nykk7brvdp61d9qh42ci2r4v7rf-file-5.45/bin
IN_NIX_SHELL=pure
doInstallCheck=
HOME=/Users/robertrichter
NIX_BINTOOLS=/nix/store/q4ppvkmxsgq83xrpsxvq7d8pp7ldz8z7-cctools-binutils-darwin-wrapper-16.0.6-973.0.1
NIX_DONT_SET_RPATH=1
depsTargetTargetPropagated=
cmakeFlags=
NIX_SSL_CERT_FILE=/nix/var/nix/profiles/default/etc/ssl/certs/ca-bundle.crt
version=0.0.1
LD_DYLD_PATH=/usr/lib/dyld
outputs=out
NIX_STORE=/nix/store
TMPDIR=/tmp
CMAKE_PREFIX_PATH=/nix/store/9g8qxz6r8x7g01hkzy739m5j6bawk4d5-clang-wrapper-16.0.6:/nix/store/q4ppvkmxsgq83xrpsxvq7d8pp7ldz8z7-cctools-binutils-darwin-wrapper-16.0.6-973.0.1:/nix/store/18jgss89rcz1ysldhrvm7m9lbcj768gc-autoconf-2.72:/nix/store/6n71i6is8mfi9qhy4zyxchlxf43mkhw3-cmake-3.29.3:/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9:/nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev:/nix/store/5wjvy31xkn743cbdwhdqpwprykl4hvxj-sqlite-3.45.3-bin:/nix/store/8dgcwdq4c49ic9mwxgnshjq2skcw7wgq-sqlite-3.45.3:/nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev:/nix/store/v9m6cwxrhpxjdc5azj20xm7rpmjicv8h-lld-16.0.6-lib:/nix/store/qpx7h1mkpd2sddrn378cvsn414nyxcn0-lld-16.0.6:/nix/store/0rvnpj6qh3xyjkf538wgsrhqnmg8nqi0-ninja-1.12.1:/nix/store/h7mvxwg6iz7gi5b6s0xvkvvrzn9c4ync-parallel-20240522:/nix/store/47zwbmparr2g3b0hci44vma95k1j8a3v-coreutils-9.5:/nix/store/mbhpg3wchwrikial8y7jhqlqv16g3vwi-dos2unix-7.5.2:/nix/store/9vj68x7aqrks90kpxx5mnnab6h28p17c-python3.11-gcovr-7.2:/nix/store/28h6zqrisbkrj8nwidv1j2zlhz19k0f9-python3.11-colorlog-6.8.2:/nix/store/crn5yw4jif2mpflc92w76piph5d1523r-python3.11-jinja2-3.1.4:/nix/store/bf3y0lxhad0g6mbvam677nlz8jry059m-python3.11-markupsafe-2.1.5:/nix/store/dwys9bb7f66hmh0rqf835f0x3xkk4kgw-python3.11-lxml-5.1.0:/nix/store/z0jcha2akzx6hc1fs9bb02glf7w89grs-python3.11-pygments-2.17.2:/nix/store/9dyk4r1yq1r6nhc26q70z1bpqh9cfarp-python3.11-pexpect-4.9.0:/nix/store/hjp93nsf2p3wkw8j9bd7a2sn4vqgxv7f-python3.11-ptyprocess-0.7.0:/nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev:/nix/store/83al8wvjjmhdwj7zl1al6x1nkjc39bxg-tbb-2021.11.0:/nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev:/nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev:/nix/store/s83f0jp0fvapgzla0acn4z4vav94jhsb-ncurses-6.4.20221231:/nix/store/j48yyijyi65472a9dybgagp0iy1syvzb-readline-8.2p10:/nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev:/nix/store/9x6isx70p9syn63by9wb9f50548fyzgv-boost-1.81.0:/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3:/nix/store/r6kgpxp1ap8h28wr60qlxzy02ijj279d-postgresql-16.3-lib:/nix/store/4wcl7yi1qdq1xd4nxqv8k5q4d9kyrkxg-update-autotools-gnu-config-scripts-hook:/nix/store/7n8zp0cpbnf1rf7d3dc7gi0x43iadqn3-sdkroot-11.0:/nix/store/mhmq1fci2kw6jp6qjv9hnryx147d8zpr-libcxx-16.0.6-dev:/nix/store/3ba5688fvifsfv844f711xyzv6kqh9mn-libcxx-16.0.6:/nix/store/flagrdvds9knmjba8c4r9wqhj46s7xdb-compiler-rt-libc-16.0.6-dev:/nix/store/50mr1scw08hh8camsc8yd594167m5sk6-compiler-rt-libc-16.0.6:/nix/store/9rgb0lr5i4vbizzhac5b6fk169n8mrkk-apple-framework-CoreFoundation-11.0.0:/nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0
LD=ld
doCheck=
NIX_DONT_SET_RPATH_FOR_BUILD=1
__propagatedImpureHostDeps=
depsBuildBuild=
PYTHONPATH=/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/lib/python3.11/site-packages:/nix/store/9vj68x7aqrks90kpxx5mnnab6h28p17c-python3.11-gcovr-7.2/lib/python3.11/site-packages:/nix/store/28h6zqrisbkrj8nwidv1j2zlhz19k0f9-python3.11-colorlog-6.8.2/lib/python3.11/site-packages:/nix/store/crn5yw4jif2mpflc92w76piph5d1523r-python3.11-jinja2-3.1.4/lib/python3.11/site-packages:/nix/store/bf3y0lxhad0g6mbvam677nlz8jry059m-python3.11-markupsafe-2.1.5/lib/python3.11/site-packages:/nix/store/dwys9bb7f66hmh0rqf835f0x3xkk4kgw-python3.11-lxml-5.1.0/lib/python3.11/site-packages:/nix/store/z0jcha2akzx6hc1fs9bb02glf7w89grs-python3.11-pygments-2.17.2/lib/python3.11/site-packages:/nix/store/9dyk4r1yq1r6nhc26q70z1bpqh9cfarp-python3.11-pexpect-4.9.0/lib/python3.11/site-packages:/nix/store/hjp93nsf2p3wkw8j9bd7a2sn4vqgxv7f-python3.11-ptyprocess-0.7.0/lib/python3.11/site-packages
TERM=tmux-256color
NIX_NO_SELF_RPATH=1
PATH_LOCALE=/nix/store/aaspx7ivx2b09cl6aqrcm4m19f2c598x-adv_cmds-119-locale/share/locale
SIZE=size
propagatedNativeBuildInputs=
strictDeps=
USER=robertrichter
AR=ar
AS=as
NIX_CC_WRAPPER_TARGET_HOST_aarch64_apple_darwin=1
TEMP=/tmp
SHLVL=3
NIX_BUILD_TOP=/tmp
NM=nm
PAGER=less
NIX_CFLAGS_COMPILE= -frandom-seed=w5cy0vgfa1 -isysroot /nix/store/7n8zp0cpbnf1rf7d3dc7gi0x43iadqn3-sdkroot-11.0/MacOSX11.0.sdk -isystem /nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/include -isystem /nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/include -isystem /nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev/include -isystem /nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev/include -isystem /nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev/include -isystem /nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev/include -isystem /nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev/include -isystem /nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev/include -isystem /nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev/include -isystem /nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev/include -isystem /nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev/include -isystem /nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev/include -isystem /nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev/include -isystem /nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev/include -isystem /nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/include -isystem /nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/include -isystem /nix/store/mhmq1fci2kw6jp6qjv9hnryx147d8zpr-libcxx-16.0.6-dev/include -isystem /nix/store/mhmq1fci2kw6jp6qjv9hnryx147d8zpr-libcxx-16.0.6-dev/include -isystem /nix/store/flagrdvds9knmjba8c4r9wqhj46s7xdb-compiler-rt-libc-16.0.6-dev/include -isystem /nix/store/flagrdvds9knmjba8c4r9wqhj46s7xdb-compiler-rt-libc-16.0.6-dev/include -iframework /nix/store/9rgb0lr5i4vbizzhac5b6fk169n8mrkk-apple-framework-CoreFoundation-11.0.0/Library/Frameworks -iframework /nix/store/9rgb0lr5i4vbizzhac5b6fk169n8mrkk-apple-framework-CoreFoundation-11.0.0/Library/Frameworks -isystem /nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/include -isystem /nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/include -isystem /nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/include -isystem /nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/include -isystem /nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev/include -isystem /nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev/include -isystem /nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev/include -isystem /nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev/include -isystem /nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev/include -isystem /nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev/include -isystem /nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev/include -isystem /nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev/include -isystem /nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev/include -isystem /nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev/include -isystem /nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev/include -isystem /nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev/include -isystem /nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/include -isystem /nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/include -isystem /nix/store/mhmq1fci2kw6jp6qjv9hnryx147d8zpr-libcxx-16.0.6-dev/include -isystem /nix/store/mhmq1fci2kw6jp6qjv9hnryx147d8zpr-libcxx-16.0.6-dev/include -isystem /nix/store/flagrdvds9knmjba8c4r9wqhj46s7xdb-compiler-rt-libc-16.0.6-dev/include -isystem /nix/store/flagrdvds9knmjba8c4r9wqhj46s7xdb-compiler-rt-libc-16.0.6-dev/include -iframework /nix/store/9rgb0lr5i4vbizzhac5b6fk169n8mrkk-apple-framework-CoreFoundation-11.0.0/Library/Frameworks -iframework /nix/store/9rgb0lr5i4vbizzhac5b6fk169n8mrkk-apple-framework-CoreFoundation-11.0.0/Library/Frameworks -isystem /nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/include -isystem /nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/include
__impureHostDeps=/bin/sh /usr/lib/libSystem.B.dylib /usr/lib/system/libunc.dylib /dev/zero /dev/random /dev/urandom /bin/sh
patches=
ZERO_AR_DATE=1
NIX_IGNORE_LD_THROUGH_GCC=1
buildInputs=
SSL_CERT_FILE=/no-cert-file.crt
depsBuildTarget=
DETERMINISTIC_BUILD=1
out=/nix/store/w5cy0vgfa1883sxxlgrbspmfyx3bplkw-hyrise
STRIP=strip
XDG_DATA_DIRS=/nix/store/18jgss89rcz1ysldhrvm7m9lbcj768gc-autoconf-2.72/share:/nix/store/6n71i6is8mfi9qhy4zyxchlxf43mkhw3-cmake-3.29.3/share:/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/share:/nix/store/5wjvy31xkn743cbdwhdqpwprykl4hvxj-sqlite-3.45.3-bin/share:/nix/store/0rvnpj6qh3xyjkf538wgsrhqnmg8nqi0-ninja-1.12.1/share:/nix/store/h7mvxwg6iz7gi5b6s0xvkvvrzn9c4ync-parallel-20240522/share:/nix/store/mbhpg3wchwrikial8y7jhqlqv16g3vwi-dos2unix-7.5.2/share:/nix/store/83al8wvjjmhdwj7zl1al6x1nkjc39bxg-tbb-2021.11.0/share:/nix/store/s83f0jp0fvapgzla0acn4z4vav94jhsb-ncurses-6.4.20221231/share:/nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev/share:/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/share:/usr/local/share:/usr/share:/Users/robertrichter/.nix-profile/share:/nix/var/nix/profiles/default/share
TMP=/tmp
PATH=/nix/store/h4fzifvim9ls3v6jf2l41xsd0mn69pzw-bash-interactive-5.2p26/bin:/nix/store/9g8qxz6r8x7g01hkzy739m5j6bawk4d5-clang-wrapper-16.0.6/bin:/nix/store/xhr7jq4wwh0xil8syxx8r3s20m222qp9-clang-16.0.6/bin:/nix/store/47zwbmparr2g3b0hci44vma95k1j8a3v-coreutils-9.5/bin:/nix/store/q4ppvkmxsgq83xrpsxvq7d8pp7ldz8z7-cctools-binutils-darwin-wrapper-16.0.6-973.0.1/bin:/nix/store/i6f0g5q8z8nnxiz1fi32kan4qzpgr4wv-cctools-binutils-darwin-16.0.6-973.0.1/bin:/nix/store/18jgss89rcz1ysldhrvm7m9lbcj768gc-autoconf-2.72/bin:/nix/store/6n71i6is8mfi9qhy4zyxchlxf43mkhw3-cmake-3.29.3/bin:/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/bin:/nix/store/5wjvy31xkn743cbdwhdqpwprykl4hvxj-sqlite-3.45.3-bin/bin:/nix/store/qpx7h1mkpd2sddrn378cvsn414nyxcn0-lld-16.0.6/bin:/nix/store/0rvnpj6qh3xyjkf538wgsrhqnmg8nqi0-ninja-1.12.1/bin:/nix/store/h7mvxwg6iz7gi5b6s0xvkvvrzn9c4ync-parallel-20240522/bin:/nix/store/mbhpg3wchwrikial8y7jhqlqv16g3vwi-dos2unix-7.5.2/bin:/nix/store/9vj68x7aqrks90kpxx5mnnab6h28p17c-python3.11-gcovr-7.2/bin:/nix/store/z0jcha2akzx6hc1fs9bb02glf7w89grs-python3.11-pygments-2.17.2/bin:/nix/store/s8vayg053c92j50r2ymh4xj8x9qjf0h9-ncurses-6.4.20221231-dev/bin:/nix/store/s83f0jp0fvapgzla0acn4z4vav94jhsb-ncurses-6.4.20221231/bin:/nix/store/j48yyijyi65472a9dybgagp0iy1syvzb-readline-8.2p10/bin:/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/bin:/nix/store/47zwbmparr2g3b0hci44vma95k1j8a3v-coreutils-9.5/bin:/nix/store/dj4z6gimsxzlw0b4dnpcgv4b6wxijx13-findutils-4.9.0/bin:/nix/store/7cjp56110icrph79302pmvv980c6cxw1-diffutils-3.10/bin:/nix/store/qxhcgmw2va7wdqz551bg1wn2h0m5ggy4-gnused-4.9/bin:/nix/store/9rf817iskvk4m8bph69q01q5pqbzc551-gnugrep-3.11/bin:/nix/store/i6gxar94g2iiid6v54hlnz9pipp6p950-gawk-5.2.2/bin:/nix/store/vd6i09s9gbb6d9dgvcwkhd7h680laayv-gnutar-1.35/bin:/nix/store/dl9rjapvkcnypn01cvpbns1xxi4kp5qq-gzip-1.13/bin:/nix/store/26a0s1d6vlbpvg1x59q3jjgdg51i99kc-bzip2-1.0.8-bin/bin:/nix/store/qkkliqj6qhzlkchlkqc6n2iry22gi78q-gnumake-4.4.1/bin:/nix/store/s30jrpgav677fpc9yvkqsib70xfmx7xi-bash-5.2p26/bin:/nix/store/q0c9g6cywh0n40vapnl2f4s0bq4fkh59-patch-2.7.6/bin:/nix/store/q040nslh604a99yvk3qa1avgw5sgmzda-xz-5.4.6-bin/bin:/nix/store/xy7w9nykk7brvdp61d9qh42ci2r4v7rf-file-5.45/bin
propagatedBuildInputs=
CC=clang
NIX_CC=/nix/store/9g8qxz6r8x7g01hkzy739m5j6bawk4d5-clang-wrapper-16.0.6
__ETC_PROFILE_SOURCED=1
NIX_BINTOOLS_WRAPPER_TARGET_HOST_aarch64_apple_darwin=1
depsBuildTargetPropagated=
depsBuildBuildPropagated=
CONFIG_SHELL=/nix/store/s30jrpgav677fpc9yvkqsib70xfmx7xi-bash-5.2p26/bin/bash
__structuredAttrs=
RANLIB=ranlib
NIX_HARDENING_ENABLE=bindnow pic relro stackprotector strictoverflow
__darwinAllowLocalNetworking=
NIX_LDFLAGS= -L/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/lib -L/nix/store/8dgcwdq4c49ic9mwxgnshjq2skcw7wgq-sqlite-3.45.3/lib -L/nix/store/v9m6cwxrhpxjdc5azj20xm7rpmjicv8h-lld-16.0.6-lib/lib -L/nix/store/83al8wvjjmhdwj7zl1al6x1nkjc39bxg-tbb-2021.11.0/lib -L/nix/store/s83f0jp0fvapgzla0acn4z4vav94jhsb-ncurses-6.4.20221231/lib -L/nix/store/j48yyijyi65472a9dybgagp0iy1syvzb-readline-8.2p10/lib -L/nix/store/9x6isx70p9syn63by9wb9f50548fyzgv-boost-1.81.0/lib -L/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/lib -L/nix/store/r6kgpxp1ap8h28wr60qlxzy02ijj279d-postgresql-16.3-lib/lib -L/nix/store/3ba5688fvifsfv844f711xyzv6kqh9mn-libcxx-16.0.6/lib -L/nix/store/50mr1scw08hh8camsc8yd594167m5sk6-compiler-rt-libc-16.0.6/lib -L/nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/lib -L/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/lib -L/nix/store/8dgcwdq4c49ic9mwxgnshjq2skcw7wgq-sqlite-3.45.3/lib -L/nix/store/v9m6cwxrhpxjdc5azj20xm7rpmjicv8h-lld-16.0.6-lib/lib -L/nix/store/83al8wvjjmhdwj7zl1al6x1nkjc39bxg-tbb-2021.11.0/lib -L/nix/store/s83f0jp0fvapgzla0acn4z4vav94jhsb-ncurses-6.4.20221231/lib -L/nix/store/j48yyijyi65472a9dybgagp0iy1syvzb-readline-8.2p10/lib -L/nix/store/9x6isx70p9syn63by9wb9f50548fyzgv-boost-1.81.0/lib -L/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/lib -L/nix/store/r6kgpxp1ap8h28wr60qlxzy02ijj279d-postgresql-16.3-lib/lib -L/nix/store/3ba5688fvifsfv844f711xyzv6kqh9mn-libcxx-16.0.6/lib -L/nix/store/50mr1scw08hh8camsc8yd594167m5sk6-compiler-rt-libc-16.0.6/lib -L/nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/lib
nativeBuildInputs=/nix/store/9g8qxz6r8x7g01hkzy739m5j6bawk4d5-clang-wrapper-16.0.6 /nix/store/18jgss89rcz1ysldhrvm7m9lbcj768gc-autoconf-2.72 /nix/store/6n71i6is8mfi9qhy4zyxchlxf43mkhw3-cmake-3.29.3 /nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9 /nix/store/fka8rjljzvqq09xnv3pr742y1mix1whi-sqlite-3.45.3-dev /nix/store/irs4zpbvbnf3m18m43wjccj8d1779fqp-lld-16.0.6-dev /nix/store/0rvnpj6qh3xyjkf538wgsrhqnmg8nqi0-ninja-1.12.1 /nix/store/h7mvxwg6iz7gi5b6s0xvkvvrzn9c4ync-parallel-20240522 /nix/store/47zwbmparr2g3b0hci44vma95k1j8a3v-coreutils-9.5 /nix/store/mbhpg3wchwrikial8y7jhqlqv16g3vwi-dos2unix-7.5.2 /nix/store/9vj68x7aqrks90kpxx5mnnab6h28p17c-python3.11-gcovr-7.2 /nix/store/9dyk4r1yq1r6nhc26q70z1bpqh9cfarp-python3.11-pexpect-4.9.0 /nix/store/1xqwb4d2nxxm4scgigag9ppnk04xalib-tbb-2021.11.0-dev /nix/store/8zxivrw650ggyr6mmrhk3q1vll4wxkwy-readline-8.2p10-dev /nix/store/2vhdf2fc5w4x4zk27nr004qbdi9fls66-boost-1.81.0-dev /nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3
CMAKE_LIBRARY_PATH=/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/lib:/nix/store/8dgcwdq4c49ic9mwxgnshjq2skcw7wgq-sqlite-3.45.3/lib:/nix/store/v9m6cwxrhpxjdc5azj20xm7rpmjicv8h-lld-16.0.6-lib/lib:/nix/store/83al8wvjjmhdwj7zl1al6x1nkjc39bxg-tbb-2021.11.0/lib:/nix/store/s83f0jp0fvapgzla0acn4z4vav94jhsb-ncurses-6.4.20221231/lib:/nix/store/j48yyijyi65472a9dybgagp0iy1syvzb-readline-8.2p10/lib:/nix/store/9x6isx70p9syn63by9wb9f50548fyzgv-boost-1.81.0/lib:/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/lib:/nix/store/r6kgpxp1ap8h28wr60qlxzy02ijj279d-postgresql-16.3-lib/lib:/nix/store/3ba5688fvifsfv844f711xyzv6kqh9mn-libcxx-16.0.6/lib:/nix/store/50mr1scw08hh8camsc8yd594167m5sk6-compiler-rt-libc-16.0.6/lib:/nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/lib:/nix/store/327bf08j5b7l9cnzink3g4vp32y5352j-python3-3.11.9/lib:/nix/store/8dgcwdq4c49ic9mwxgnshjq2skcw7wgq-sqlite-3.45.3/lib:/nix/store/v9m6cwxrhpxjdc5azj20xm7rpmjicv8h-lld-16.0.6-lib/lib:/nix/store/83al8wvjjmhdwj7zl1al6x1nkjc39bxg-tbb-2021.11.0/lib:/nix/store/s83f0jp0fvapgzla0acn4z4vav94jhsb-ncurses-6.4.20221231/lib:/nix/store/j48yyijyi65472a9dybgagp0iy1syvzb-readline-8.2p10/lib:/nix/store/9x6isx70p9syn63by9wb9f50548fyzgv-boost-1.81.0/lib:/nix/store/vl3pk26iqibzi5k963w5lzr1h66bz0p8-postgresql-16.3/lib:/nix/store/r6kgpxp1ap8h28wr60qlxzy02ijj279d-postgresql-16.3-lib/lib:/nix/store/3ba5688fvifsfv844f711xyzv6kqh9mn-libcxx-16.0.6/lib:/nix/store/50mr1scw08hh8camsc8yd594167m5sk6-compiler-rt-libc-16.0.6/lib:/nix/store/5rzipngaarjfcjbgz6faqxay6sivvcc3-libobjc-11.0.0/lib
name=hyrise
depsHostHostPropagated=
_=/nix/store/47zwbmparr2g3b0hci44vma95k1j8a3v-coreutils-9.5/bin/env
```

</details>